### ISSUE TITLE
wrong line number reported when tab characters are detected and code begins with comments

### ISSUE THREAD

**Issue by jackwasey** [jimhester/lintr#134]
It seems to be that it doesn't count initial comment lines. It seems like github might remove the critical tab in the following code block:

`/tmp/tab.R`

```
# comments
# comments
# comments
# comments
# comments
# comments
# comments

some_function <- function() {
        print("this line preceded by tab")
}

```

then 
`lintr::lint("/tmp/tab.R")`

Giving me:

```
tab.R:2:1: style: Use two spaces to indent, never tabs.
        print("this line preceded by tab")
```

**Comment by fangly** [comment]
Thank you jackwasey. There was indeed an issue with no_tab_linter() not handling line numbers correctly. This manifested when there were commented lines in the content to lint. This is fixed in 65b312f  .
